### ISSUE TITLE
what is the best hyperparameters

### ISSUE THREAD

**Issue by CSyyn** [wzhwzhwzh0921/S-D-Mamba#8]
I run the default scripts for weather dataset with input 96 and predict 96, but I get the test results are quite different from paper, could you please tell me the best hyperparameters corresponding to the best result in your paper, thanks~
 
my results
![image](https://github.com/wzhwzhwzh0921/S-D-Mamba/assets/63851708/890f7982-8009-4e1c-a1b3-fb824127e26d)

paper results
![image](https://github.com/wzhwzhwzh0921/S-D-Mamba/assets/63851708/9b0f6dec-c0b2-42f9-8459-957506b46a8e)

**Comment by wzhwzhwzh0921** [comment]
the best hyperparameters is already in the scripts

**Comment by CSyyn** [comment]
I run another scripts, still get quite different results from your paper

**Comment by wzhwzhwzh0921** [comment]
Can you send the log of your training? So i can check~

**Comment by wzhwzhwzh0921** [comment]
I just verified it. I created a new environment and tested the weather data set on the new environment（T=96，S=96）. This is my training log, and the effect is in line with the results we gave. I noticed that you also encountered a problem that cannot be reproduced when reproducing timeMachine. Maybe there is a problem with the mamba package you installed? 
[Uploading weather96_96(24.4.8).txt…]()

**Comment by wzhwzhwzh0921** [comment]
I just verified it. I created a new environment and tested the weather data set on the new environment（T=96，S=96）. This is my training log, and the effect is in line with the results we gave. I noticed that you also encountered a problem that cannot be reproduced when reproducing timeMachine. Maybe there is a problem with the mamba package you installed?
[weather96_96(24.4.8).txt](https://github.com/wzhwzhwzh0921/S-D-Mamba/files/14905326/weather96_96.24.4.8.txt)
